.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: dart
+sudo: false
+
+# Run against dev and stable channels.
+dart:
+  - dev
+  - stable
+# Define test tasks to run.
+dart_task:
+  # Run the tests -- include the default-skipped presubmit tests.
+  - test: --platform vm --run-skipped
+  - dartfmt
+  - dartanalyzer
```

Comment:
In https://github.com/dart-lang/oss_project_templates/blob/master/dart-github-dart-lang/.travis.yml, the dartfmt and dartanalyzer tasks are only run once per platform (dev/stable). Do we want to do the same here?